build-image/first-boot.bash

```diff
@@ -158,10 +158,11 @@ if hash python3 2>/dev/null; then bash /boot/webif.bash reinsure_running; fi
 echo -n "$(timestamp) [openHABian] Installing git package... "
 if apt-get install --yes git &>/dev/null; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
 
-if [ -d /opt/openhabian/ ]; then cd /opt && rm -rf /opt/openhabian/; fi
+if [ -d /opt/openhabian ]; then cd /opt && rm -rf /opt/openhabian; fi
 # shellcheck disable=SC2154
 echo -n "$(timestamp) [openHABian] Cloning myself from ${repositoryurl}, ${clonebranch} branch... "
-if git clone -q -b "$clonebranch" "$repositoryurl" /opt/openhabian; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+if ! git clone -q -b "$clonebranch" "$repositoryurl" /opt/openhabian; then echo "FAILED"; fail_inprogress; fi
```

Comment:
Personally, I prefer the other style of fail checking where the ok is included in the check.

Reply:
me too but it doesn't work here when we want clean output

Reply:
Oh, yeah, I forgot that git would screw that up. Don't you need to have echo print the newline so that git doesn't start printing on the same line?